.eslintrc

```diff
@@ -7,6 +7,7 @@ globals:
   browser: true
   context: true
   jestPuppeteer: true
+  BASE_URL: true
```

Comment:
@HashTalmiz this line (and many others) that are supposedly changed in the PR have the exact same content as in master, which makes me think that something (perhaps the editor you're using) is adding different line ending characters (or something else invisible like tabs vs spaces) to all the lines that you edit (including the ones you merged) so that they show up different in diffs even though nothing has really changed with them.

Maybe you can test this by doing local `git diff` after saving with different editors (though you may have to explicitly remove special line ending characters if they are in there now).

Reply:
The thing is, I haven't made any changes manually to any of these files for my editor to do anything (Except for webpack.config and package.json) All of these were bought by the rebase which I tried to do with the master branch

Reply:
If it is comparing to master, I'm really not sure why its showing a difference, is there a possibility that I did something wrong with the rebase?

Reply:
Ah I found the culprit, opening a review there